courses/machine_learning/deepdive2/structured/labs/4b_keras_dnn_babyweight.ipynb

```diff
@@ -0,0 +1,574 @@
+{
```

Comment:
<p>why not combine these three commands to a single</p><p>%%bash</p><p>head -5 *.csv</p><p> </p><p>I think it still shows the result fairly concisely. Just personal preference.</p><p> </p><p>If changed, adjust in solution notebook as well.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/672/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Ya, that's fair. Less cells to run and exactly the same output. I just wanted to anyways to be a simple, "Look everything is there, yay!" thing which this will do too. Changed.</p>

Reply:
ok.